src/Serilog.Sinks.Fluentd/Sinks/Fluentd/FluentdSinkOptions.cs

```diff
@@ -41,6 +43,8 @@ protected FluentdSinkOptions()
             Period = TimeSpan.FromSeconds(2);
             Tag = "Tag";
             MessageTemplateKey = "mt";
+            UseUnixDomainSocketEndpoit = false;
+            UdsSocketFilePath = String.Empty;
```

Comment:
If uds options are provided in default constructor,
I assume that host/port also should receive default values.

Reply:
They're implicitly being set in `LoggerConfigurationFluentdExtensions` but I think it's a good idea to have them here as well.